tt_metal/host_api.hpp

```diff
@@ -26,16 +20,21 @@
  * https://www.tablesgenerator.com/markdown_tables
  * */
 
+class CoreRange;
+class CoreRangeSet;
```

Comment:
I wonder how hard it would be to pull these in the `tt::tt_metal` namespace, maybe we can do that when we pull umd in since they use `CoreCoord` which in turn uses umd's `tt_xy_pair`

Reply:
yes that should be done, having this live in global namespace is weird